modules/examples/flake/flake.nix

```diff
@@ -3,7 +3,7 @@
 
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
-    nix-darwin.url = "github:LnL7/nix-darwin";
+    nix-darwin.url = "github:LnL7/nix-darwin/master";
```

Comment:
Hello,
Why this change? Isn't it already the default 🤔

Reply:
Yeah. The `eval-config.nix` error tells people to ensure their input looks like `github:LnL7/nix-darwin/NIX-DARWIN-BRANCH`, and it felt simpler to just align the examples with that. I don’t love how it looks but I couldn’t think of a way to account for it in the `eval-config.nix` error that wouldn’t confuse people or be even more verbose. But I’m open to alternative suggestions.

My ideal would be to rename our `master` branch to `nix-darwin-unstable` to match `nixpkgs-unstable`, but that requires repo admin privileges so it’s not easy to do immediately.